src/explorer/Metrics/UntypedMetric.cs

```diff
@@ -1,3 +1,5 @@
+#pragma warning disable CA1815 // Struct type should override Equals
+
```

Comment:
Maybe it makes more sense to change `UntypedMetric` into a class instead of a struct? The `Metric` member is almost always a reference type anyway...

Reply:
fixed with 1f0470d